fibonacci/py/code.py

```diff
@@ -1,6 +1,7 @@
 import sys
+from functools import cache
 
-
+@cache
```

Comment:
Using a cache is "cheating" in this case. It allows the program to skip huge amounts of the work relative to other implementations. I mean, it's cool that python supports this, but a similar cache could be implemented in other languages giving them huge boosts also.

Reply:
Ok, wouldn't you have to disable all JIT and compiler optimizations too, because they are essentially "cheating" too? Otherwise the benchmark is not comparing *languages* but compilers.